src/main/java/wedlog/address/logic/commands/RedoCommand.java

```diff
@@ -0,0 +1,40 @@
+package wedlog.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static wedlog.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import wedlog.address.logic.commands.exceptions.CommandException;
+import wedlog.address.model.Model;
+
+//@@author samuelim01-reused
+// Reused from Address Book(Level 4) with minor modifications.
+
+/**
+ * Restores the {@code Model}'s address book to its previous state before undo.
+ */
+public class RedoCommand extends Command {
+
+    public static final String COMMAND_WORD = "redo";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Undoes the most recent change before undo to WedLog. ";
```

Comment:
Perhaps a clearer way to describe this would be "Redoes the last action that was undone" (from the UG), or "Reverses the most recent undo command"?

Reply:
I think "Reverses the most recent undo command" sounds better here.